satellite/console/usagerollups.go

```diff
@@ -25,3 +26,25 @@ type ProjectUsage struct {
 	Since  time.Time
 	Before time.Time
 }
+
+// BucketUsageRollup is total bucket usage info
+// for certain period
+type BucketUsageRollup struct {
+	ProjectID  uuid.UUID
+	BucketName []byte
+
+	RemoteStoredData float64
```

Comment:
Why all `float64`?

Reply:
because we are displaying these data in `GB` or `GB*hours`